File: py/sorts/bubblesort.py
def bubblesort(l):
    last = l[:]
    for i in range(0, len(l))-1):
        for i in range(1, len(l)):
            if l[i-1] > l[i]:
                l[i-1], l[i] = l[i], l[i-1]
        if last != l:
            last = l[:]
        else:
            break
    return l

File: cs/sorts/bubblesort.cs
using System.Collections.Generic;

namespace bubblesort
{
    class Program
    {
        static List<int> Bubblesort(List<int> array)
        {
            int tmp;
            if (array.Count > 1)
            {
                for (int i=0; i < array.Count; i++)
                {
                    for (int j=1; j < array.Count; j++)
                    {
                        if (array[j-1] > array[j]) {
                            tmp = array[j - 1];
                            array[j - 1] = array[j];
                            array[j] = tmp;
                        }
                    }
                }
            }
            return array;
        }
    }
}
